### ISSUE TITLE
get_gsod returns additional station data

### ISSUE THREAD

**Issue by EFreiberger** [ropensci/GSODR#49]
I tried to download data from 2 stations with this piece of code

```R
install.packages("GSODR")
library(GSODR)
library(dplyr)

load(system.file("extdata", "country_list.rda", package = "GSODR"))
load(system.file("extdata", "isd_history.rda", package = "GSODR"))

a <- get_GSOD(years = 2010, station = "489300-99999")    
b <- get_GSOD(years = 2010, station = "489260-99999")
```
I don't understand why b contains data from a:

![image](https://user-images.githubusercontent.com/46746047/51245077-f77d3200-1986-11e9-9792-f1455eba2d1f.png)

I used package version 1.3.1 and R version 3.5.0.

Many thanks for your help!

**Comment by adamhsparks** [comment]
Ah, hmm. I have an idea what's going on, but I don't know how or why.

Thanks for alerting me to this.

**Comment by adamhsparks** [comment]
Found it. Fixed it.

https://github.com/ropensci/GSODR/commit/5dcf94459d75713708121938332813830783d69c

I'll try to get a new release on CRAN in the next few days. Tomorrow I'll update the master branch here on GitHub.

Thanks for reporting.

**Comment by adamhsparks** [comment]
Closed with release of 1.3.2 on CRAN. https://github.com/ropensci/GSODR/releases/tag/1.3.2
